companies/target.md

```diff
@@ -0,0 +1,36 @@
+---
+fortune_rank: 37
+name: Target
+tags: ["fortune500", "retail"]
+image: target.png
+blm_statements:
+  - url: https://twitter.com/TargetNews/status/1268924275597160450
+    date_posted: 2020-06-05
+    type: Tweet
+    raw_text: Target stands with Black families, communities and team members, and we’re committed to using our size scale and resources to help heal and create lasting change.
+  - url: https://corporate.target.com/article/2020/06/commitments-rebuilding-and-social-justice
+    date_posted: 2020-06-05
+    type: Blog
+    raw_text: >
+      In a recent note, CEO Brian Cornell shared the Target team’s commitment to stand against racism. We pledged to face anxiety, fear and sorrow with purpose, and formed an enterprise task force to determine how to help create solutions for the injustice Black families and people of color face every day. 
+
+      Today, our next steps include announcing a $10 million commitment and ongoing resources to advance social justice and support rebuilding and recovery efforts in local communities. 
+
+      “Target stands with Black families, communities and team members. As we face an inflection point in Minneapolis and across the country, we’re listening to our team, guests and communities, committed to using our size, scale and resources to help heal and create lasting change,” says Brian Cornell, chairman and CEO, Target.
+
+      Initial efforts include:
+
+      A $10 million investment from Target and the Target Foundation to support long-standing partners such as the National Urban League and the African American Leadership Forum. We will also add new partners in the Minneapolis-St. Paul area and across the country. 
```

Comment:
```suggestion
      A $10 million investment from Target and the Target Foundation to support long-standing partners such as the National Urban League and the African American Leadership Forum. We will also add new partners in the Minneapolis-St. Paul area and across the country.
```

A few "trailing spaces" errors.  Just means erase the space(s) at the end of the line to clean it up :)